server/routes/api/v1/petTypesRouter.js

```diff
@@ -8,18 +10,57 @@ petTypesRouter.get('/', async (req, res) => {
     const petTypes = await PetType.findAll()
     res.status(200).json({ petTypes: petTypes })
   } catch (error) {
-    return res.status(500).json({ errors: error })
+    res.status(500).json({ errors: error })
   }
 })
 
 petTypesRouter.get('/:petType', async (req, res) => {
   try {
     const petType = await PetType.findByType(req.params.petType)
-    petType.adoptablePets = await petType.adoptablePets()
+    petType.adoptablePets = await petType.getAvailablePets()
     res.status(200).json({ pets: petType })
   } catch (error) {
-    return res.status(500).json({ errors: error })
+    res.status(500).json({ errors: error })
   }
 })
 
+petTypesRouter.get('/:petType/:adoptablePetId', async (req, res) => {
+  try {
```

Comment:
looks like the purpose of this endpoint is to view a single adoptable pet, meaning the request should be processed through an adoptable pets router, not the pet types